src/components/atoms/TextField/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export interface TextFieldProps {
+  className?: string;
+  type: string;
+  placeholder: string;
+  min?: string;
+  max?: string;
+  css?: {};
+}
+const TextField = (props: TextFieldProps) => {
+  return <input className="inp" style={props.css} {...props} />;
```

Comment:
props should be destructured
should have atleast value props

Reply:
resolved